docs/azure_arc_jumpstart/azure_arc_k8s/day2/microk8s/local_microk8s_monitor/_index.md

```diff
@@ -0,0 +1,99 @@
+
+In this guide, you will hook the microK8s cluster to Azure Monitor by deploying the [OMS agent](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/log-analytics-agent) on your Kubernetes cluster to start collecting telemetry.  
+
+> **Note: This guide assumes you already deployed MicroK8s and connected it to Azure Arc. If you haven't, this repository offers you a way to do so in the [MicroK8s onboarding guide](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/microk8s/local_microk8s/).**
+
+## Prerequisites
+
+* Clone the Azure Arc Jumpstart repository
+
+    ```shell
+    git clone https://github.com/microsoft/azure_arc.git
+    ```
+
+* [Install or update Azure CLI to version 2.15.0 and above](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest). Use the below command to check your current installed version.
+
+  ```shell
+  az --version
+  ```
+
+* Create Azure service principal (SP)
+
+    To be able to complete the scenario and its related automation, Azure service principal assigned with the “Contributor” role is required. To create it, login to your Azure account run the below command (this can also be done in [Azure Cloud Shell](https://shell.azure.com/)).
+
+    ```shell
+    az login
+    az ad sp create-for-rbac -n "<Unique SP Name>" --role contributor
+    ```
+
+    For example:
+
+    ```shell
+    az ad sp create-for-rbac -n "http://AzureArcK8s" --role contributor
+    ```
+
+    Output should look like this:
+
+    ```json
+    {
+    "appId": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "displayName": "AzureArcK8s",
+    "name": "http://AzureArcK8s",
+    "password": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "tenant": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+    }
+    ```
+
+    > **Note: It is optional but highly recommended to scope the SP to a specific [Azure subscription and resource group](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest)**
+
+## Azure Monitor for Containers Integration
+
+* Retrieve either the shell and PowerShell scripts [provided](https://github.com/microsoft/azure_arc/tree/main/azure_arc_k8s_jumpstart/microk8s/azure_monitor).
+
+* Before integrating the cluster with Azure Monitor for Containers, click on the "Insights (preview)" blade for the connected Arc cluster to show how the cluster is not currently being monitored.
+
+    ![An existing Azure Arc enabled Kubernetes cluster](./01.png)
+
+    ![An existing Azure Arc enabled Kubernetes cluster with no Azure Monitor integration](./02.png)
+
+* Edit the environment variables in either of the scripts to match your environment parameters, run it using the ```. ./microk8s_monitor_onboarding.sh``` (Bash) or ```./microk8s_monitor_onboarding.ps1``` (PowerShell) command.
+
```

Comment:
These commands will run the scripts. For the edit, provide either a screenshot or a code block as an example.

Reply:
See an example to an example :-) [in this section](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/aks/aks_arm_template/#deployment)

Reply:
Have added yesterday the section below - let me know if that works or if you prefer the image 

```shell
# <--- Change the following environment variables according to your Azure service principal name --->
echo "Exporting environment variables"
export subscriptionId='<Your Azure subscription ID>'
export appId='<Your Azure service principal name>'
export password='<Your Azure service principal password>'
export tenantId='<Your Azure tenant ID>'
export resourceGroup='<Azure resource group name>'
export arcClusterName='<The name of your k8s cluster as it will be shown in Azure Arc>'
```

Reply:
So example means that we are providing fictional values. Something like this (you can remove lines 1&2):

```shell
export subscriptionId='e73c1dbe-XXXX-XXXX-XXXX-c813757b1786'
export appId='051b9a58-XXXX-XXXX-XXXX-0e7ae1bca3fb'
export password='051b9a58-XXXX-XXXX-XXXX-0e7ae1bca3fb'
export tenantId='051b9a58-XXXX-XXXX-XXXX-0e7ae1bca3fb'
export resourceGroup='Arc-MicroK8s-Demo'
export arcClusterName='Arc-MicroK8s-Demo'
```

It's designed to increase the confidence of the reader.